README.md

```diff
@@ -56,7 +56,7 @@ to your `~/.bazelrc`, followed by the following set of options that are
 specific to your environment:
 
 ```
-build:mycluster --bes_backend=grpc://fill-in-the-frontend-service-hostname-here:8985
+build:mycluster --bes_backend=grpc://fill-in-the-event-service-hostname-here:8983
```

Comment:
For the Docker Compose based setup we renumbered the port numbers to account for the new scheduler. This is why :8985 is used here instead of :8983.

People (@Qinusty, @finn-ball) are currently working on bringing the other deployments in sync again with the Docker Compose based one. My hope is that those will also start using port 8985 from then on.

Could you please revert the change in port number? Thanks!

Reply:
done